.lintstagedrc

```diff
@@ -8,7 +8,9 @@
   ],
   "*.js": [
     "yarn run lint:css:fix",
-    "yarn run lint:js:fix",
+    "yarn run lint:js:fix"
```

Comment:
Revert this file, its problematic, looks like you have some very old changes in your PR